docs/source/how-to-manage.mdx

```diff
@@ -0,0 +1,186 @@
+# How to create and manage a repository
+
+A repository is a space for you to store your model or dataset files. This guide will show you how to:
+
+* Create and delete a repository.
+* Adjust repository visibility.
+* Use the [`Repository`] class for common Git operations like clone, branch, push, etc.
+
+If you want to create a repository on the Hub, you need to log in to your Hugging Face account:
+
+1. Log in to your Hugging Face account with the following command:
+
+   ```bash
+   huggingface-cli login
+   ```
+
+2. Alternatively, if you prefer working from a Jupyter or Colaboratory notebook, login with [`notebook_login`]:
+
+   ```python
+   >>> from huggingface_hub import notebook_login
+   >>> notebook_login()
+   ```
+
+   [`notebook_login`] will launch a widget in your notebook from which you can enter your Hugging Face credentials.
+
+## Create a repository
+
+Create an empty repository with [`create_repo`] and give it a name with the `repo_id` parameter. The `repo_id` is your namespace followed by the repository name: `{username_or_org}/{repo_name}`.
+
+```python
+>>> from huggingface_hub import create_repo
+>>> create_repo("lysandre/test-model")
+'https://huggingface.co/lysandre/test-model'
+```
+
+By default, [`create_repo`] creates a model repository. But you can use the `repo_type` parameter to specify another repository type. For example, if you want to create a dataset repository:
+
+```py
+>>> from huggingface_hub import create_repo
+>>> create_repo("lysandre/test-dataset", repo_type="dataset")
+'https://huggingface.co/lysandre/test-dataset'
+```
+
+When creating a repository, you also have the option to set your repository visibility with the `private` parameter. For example, if you want to create a private repository:
+
+```py
+>>> from huggingface_hub import create_repo
+>>> create_repo("lysandre/test-private", private=True)
+```
+
+If you want to change the repository visibility at a later time, you can use the [`update_repo_visibility`] function. 
+
+## Delete a repository
+
+Delete a repository with [`delete_repo`]. Make sure you are certain you want to delete a repository because this is an irreversible process!
+
+Specify the `repo_id` of the repository you want to delete:
+
+```py
+>>> delete_repo(repo_id=name)
+```
+
+If you aren't sure what the `repo_id` is or you want to quickly retrieve it, you can also use the [`get_full_repo_name`] function:
+
+```python
+>>> from huggingface_hub import get_full_repo_name, delete_repo
+>>> name = get_full_repo_name(repo_name)
+>>> delete_repo(repo_id=name)
```

Comment:
I'm not sure we really need to add this here since this is not a common use case really.

Reply:
Sounds good, I'll remove this bit then